File: app/src/main/java/com/babic/filip/movieshack/ui/list/PopularMovieHolder.java
package com.babic.filip.movieshack.ui.list;

import android.support.annotation.NonNull;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.babic.filip.movieshack.R;
import com.babic.filip.movieshack.common.utils.ImageUtils;
import com.babic.filip.movieshack.model.Movie;

public class PopularMovieHolder extends MovieHolder {

    public PopularMovieHolder(final View itemView) {
        super(itemView);
    }

    @Override
    void displayItem(@NonNull final Movie movie) {
        TextView moviePopularity = itemView.findViewById(R.id.moviePopularity);
        TextView movieName = itemView.findViewById(R.id.movieName);
        ImageView movieImage = itemView.findViewById(R.id.movieImage);

        moviePopularity.setText(itemView.getContext().getString(R.string.movie_popularity, movie.getPopularity()));
        movieName.setText(movie.getTitle());

        ImageUtils.loadImage(movieImage, movie.getImage());
    }
}

File: app/src/main/java/com/babic/filip/movieshack/ui/list/PopularMovieHolder.kt
package com.babic.filip.movieshack.ui.list

import android.view.View
import com.babic.filip.movieshack.R
import com.babic.filip.movieshack.common.utils.loadImage
import com.babic.filip.movieshack.model.Movie
import kotlinx.android.synthetic.main.item_movie_popular.view.*

class PopularMovieHolder(itemView: View) : MovieHolder(itemView) {

    override fun displayItem(movie: Movie) = with(itemView) {
        movie.image?.run { loadImage(movieImage, this) }

        moviePopularity.text = itemView.context.getString(R.string.movie_popularity, movie.popularity)
        movieName.text = movie.title
    }
}
